template/tcg-storage.tex

```diff
@@ -0,0 +1,50 @@
+%%
+% Copyright (c) 2017 - 2021, Pascal Wagler;
```

Comment:
Not needed since all contents of this file are new

Reply:
this is removed now.